### PR TITLE
feat: Make links in farm pair dropdown menu more descriptive

### PR DESCRIPTION
**Pull request by memoyil**
The new detail menu would look like this

Screenshot 2021-03-04 at 02 14 32

### CODE REVIEW

**Comment by memoyil** [comment]
@hachiojidev Could you please review this?

**Comment by memoyil** [comment]
@Chef-Chungus  Could you please review this?

**Review by Chef-Chungus — APPROVED** [review]

File: src/views/Farms/components/FarmTable/Actions/ActionPanel.tsx [hunk 1]
```diff
@@ -122,16 +122,15 @@ const ActionPanel: React.FunctionComponent<ActionPanelProps> = ({ details, apr,
     <Container>
       <InfoContainer>
         <StakeContainer>
-          Stake:
           <StyledLinkExternal href={`https://exchange.pancakeswap.finance/#/add/${liquidityUrlPathParts}`}>
-            {lpLabel}
+            Get {lpLabel}
```
Comment (hachiojidev) [inline src/views/Farms/components/FarmTable/Actions/ActionPanel.tsx:126]: Please add the translation to this

```
TranslateString(999, `Get ${lpLabel}`, { name: lpLabel })
```

**Review by hachiojidev — CHANGES_REQUESTED** [review]

File: src/views/Farms/components/FarmTable/Actions/ActionPanel.tsx (hunk 1 above)
Comment (memoyil) [reply src/views/Farms/components/FarmTable/Actions/ActionPanel.tsx]: done

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**